Dockerfile

```diff
@@ -1,42 +1,29 @@
-# Set the base image to Python
-FROM python:2.7.9
-MAINTAINER EclecticIQ <opentaxii@eclecticiq.com>
+FROM python:3.6-stretch AS build
+LABEL maintainer="EclecticIQ <opentaxii@eclecticiq.com>"
 
-# Create the working dir and set the working directory
-WORKDIR /
+RUN python3 -m venv /venv && /venv/bin/pip install -U pip setuptools
 
-# Install dependencies
-COPY ./requirements.txt /requirements.txt
-RUN pip install -r requirements.txt \
-    && pip install supervisor \
-    && pip install gunicorn \
-    && pip install psycopg2 \
-    && rm -f /requirements.txt
+RUN /venv/bin/pip install gunicorn psycopg2-binary
```

Comment:
are these test requirements? Why they are not in `requirements-dev.txt`? (I'll approve anyway, it's not essential)

Reply:
They are production requirements for the docker image, but they are *not* requirements that should be installed if you do `pip install opentaxii`.

Maybe we should consider version-pinning these, though. And maybe we can put them in `requirements-docker.txt` for consistency?